tests/integration/test_interaction.py

```diff
@@ -407,14 +475,35 @@ def test_keyboard_help(self):
         self.interact_with_keyboard_help(use_keyboard=True)
 
 
+@ddt
+class KeyboardAssessmentInteractionTest(BasicAssessmentInteractionTest, BaseIntegrationTest):
```

Comment:
@mtyaka I think you can remove `BaseIntegrationTest` (`BasicAssessmentInteractionTest` already pulls it in)?